### PR TITLE
Add nice error message when crashlytics action can't find its .framework

### PR DESCRIPTION
**Pull request by KrauseFx**
This happened when the user didn't have Crashlytics set up for their app

screen shot 2017-04-06 at 6 19 41 pm

### CODE REVIEW

File: fastlane/lib/fastlane/helper/crashlytics_helper.rb [hunk 1]
```diff
@@ -3,13 +3,15 @@ module Helper
     class CrashlyticsHelper
       class << self
         def discover_default_crashlytics_path
-          Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last
+          path = Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last
+          unless path
+            UI.user_error!("Couldn't find Crashlytics.framework in current directory. Make sure to add the 'Crashlytics' pod to your 'Gemfile' and run `pod update`")
```
Comment (ohayon) [inline fastlane/lib/fastlane/helper/crashlytics_helper.rb]: `Podfile` instead of `Gemfile` right?

**Review by ohayon — APPROVED** [review]
Just switch up the file name and then 🚀

File: fastlane/lib/fastlane/helper/crashlytics_helper.rb (hunk 1 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/helper/crashlytics_helper.rb]: Yess, nice catch 👍

**Review by mfurtak — APPROVED** [review]

File: fastlane/lib/fastlane/helper/crashlytics_helper.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class << self
         def discover_default_crashlytics_path
           path = Dir["./Pods/iOS/Crashlytics/Crashlytics.framework"].last || Dir["./**/Crashlytics.framework"].last
           unless path
-            UI.user_error!("Couldn't find Crashlytics.framework in current directory. Make sure to add the 'Crashlytics' pod to your 'Gemfile' and run `pod update`")
+            UI.user_error!("Couldn't find Crashlytics.framework in current directory. Make sure to add the 'Podfile' pod to your 'Gemfile' and run `pod update`")
```
Comment (ohayon) [inline fastlane/lib/fastlane/helper/crashlytics_helper.rb]: Change `Podfile` to `Crashlytics` and `Gemfile` to `Podfile` 😄

**Review by ohayon — CHANGES_REQUESTED** [review]

File: fastlane/lib/fastlane/helper/crashlytics_helper.rb (hunk 2 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/helper/crashlytics_helper.rb]: ok, I was definitely too tired to code yesterday. I'll try my best to do it right this time

**Review by ohayon — APPROVED** [review]

**Pull request merged**